models/tmp/stg_apple_store__app_crash_tmp.sql

```diff
@@ -1,12 +1,12 @@
 {{
     fivetran_utils.union_data(
-        table_identifier='app', 
+        table_identifier='app_crash_daily', 
         database_variable='apple_store_database', 
         schema_variable='apple_store_schema', 
         default_database=target.database,
         default_schema='apple_store',
```

Comment:
Based on my recent discovery, this should also be the true default schema. In this case `itunes_connect`.

Reply:
switched